packages/core/redux/persistAccount/utils.js

```diff
@@ -3,8 +3,8 @@
  * Licensed under the AGPL Version 3 license.
  */
 
-import bip39 from 'bip39'
 import uuid from 'uuid/v1'
+import { DEFAULT_PATH } from '@chronobank/core/services/signers/EthereumMemoryDevice'
```

Comment:
Please use relative path instead of abs. path. Import from '@chronobank/core' is restricted in packages/core